ui/src/admin/components/chronograf/UsersTableHeader.js

```diff
@@ -21,30 +20,35 @@ class UsersTableHeader extends Component {
         <h2 className="panel-title">
           {panelTitle} in <em>{organization.name}</em>
         </h2>
-        <Authorized requiredRole={ADMIN_ROLE}>
-          <button
-            className="btn btn-primary btn-sm"
-            onClick={onClickCreateUser}
-            disabled={isCreatingUser}
-          >
-            <span className="icon plus" />
-            Create User
-          </button>
-        </Authorized>
+        <button
```

Comment:
Was the Authorized redundant because only admins/superadmins can reach this page?

Reply:
yep